src/components/charts/LinearGauge/LinearGaugeProgress.tsx

```diff
@@ -37,24 +44,34 @@ export const LinearGaugeProgress = ({
 
 const BackgroundCircle = styled.circle<{
   strokeWidth: number
+  topGap: number
+  circleStartPoint: number
 }>`
   fill: none;
   stroke: ${({ theme }) => theme.colors.background.tertiary};
   stroke-width: ${({ strokeWidth }) => strokeWidth};
+  stroke-dasharray: 100, 100;
+  stroke-dashoffset: ${({ topGap }) => `${topGap}`};
+  transform: ${({ circleStartPoint }) => `rotate(${circleStartPoint}deg)`};
+  transform-origin: center;
 `
 
 const ProgressCircle = styled.circle<{
   color: ContentColor
   percentage: number
   strokeWidth: number
+  topGap: number
+  circleStartPoint: number
 }>`
   fill: none;
-  stroke-dasharray: ${({ percentage }) => `${percentage},${100}`};
+  stroke-dasharray: ${({ percentage, topGap }) =>
+    `${percentage + topGap / 2 > 0 ? percentage + topGap / 2 : 0},${100}`};
+  stroke-dashoffset: ${({ topGap }) => `${topGap}`};
```

Comment:
Now you don't need to send these as props anymore, but can use them directly 🙂 

```suggestion
  stroke-dashoffset: ${TOP_GAP};
```